test/features/kogito_service_performance.feature

```diff
@@ -144,13 +143,12 @@ Feature: Kogito Service Performance
     And Infinispan instance "external-infinispan" is deployed for performance within 5 minute(s) with configuration:
       | username | developer |
       | password | mypass    |
+    And Install Infinispan Kogito Infra "external-infinispan" connected to resource "external-infinispan" within 5 minutes
```

Comment:
This KogitoInfra is deployed instantly so we don't have to wait for its deployment?

Reply:
That's correct, it's a CR to bind third party CRs with our Kogito services.

Reply:
The test is waiting here for KogitoInfra to be reconciled first. Otherwise the KogitoInfra status will be empty when KogitoRuntime is created and no properties/env variables are applied. I kept this to be consistent with case when infrastructure is created by KogitoInfra.

Reply:
Right, I didn't notice that it is external infinispan which we already have running so we don't have to wait here for the pod to be running.